astro/src/content/blog/amazon-cognito-and-fusionauth-comparison.mdx

```diff
@@ -68,7 +68,7 @@ If you would like more information on how FusionAuth enables effective EIU acros
 ### Custom Feature Development
 Another difference between Cognito and FusionAuth is our size and flexibility. There is no doubt that Amazon is a huge corporation that reaches all corners of the planet. At their size, it is difficult to get custom features or functionality unless they can serve their all Cognito customers. Smaller companies with unique or non-standard needs will find it difficult to get any type of custom feature implemented.
 
-FusionAuth is a small, bootstrapped company dedicated to our customers. We have successfully provided our core solutions with unique customizations and one-on-one support for our clients. We have eliminated the overhead, complex approval pipelines, and corporate red-tape that cripples large firms so we can deliver exceptional solutions for clients like DataStax, StrategyCorp, Deutsche Bank and IBM. If you have a specific challenge for managing users that we don't already cover, we'll work with you for a solution. This is an added benefit to having a single-tenant solution: your system can be customized to fit your specific requirements and specifications.
+FusionAuth is dedicated to our customers. We have successfully provided our core solutions with unique customizations and one-on-one support for our clients. We have eliminated the overhead, complex approval pipelines, and corporate red-tape that cripples large firms so we can deliver exceptional solutions for clients like DataStax, StrategyCorp, Deutsche Bank and IBM. If you have a specific challenge for managing users that we don't already cover, we'll work with you for a solution. This is an added benefit to having a single-tenant solution: your system can be customized to fit your specific requirements and specifications.
```

Comment:
I think it's fair to say that we still do (see: Life360) but I don't know that it's something that we want to advertise anymore at this point.

Though I suppose it could be argued that Life360 isn't custom development as much as it is a "hurry up" on Issue #1